src/commands/Suggestion Commands/AcceptCommand.ts

```diff
@@ -64,6 +64,20 @@ export const run: RunFunction = async (client, message, args) => {
 			color: 'GREEN',
 		})
 	);
+	
+const author: User = client.users.cache.get((Suggestion as Anything).User);
+	if (author) {
+		author.send(
+			new MessageEmbed({
+				name: 'Your suggestion was accepted!',
+				value: `A staff member accepted your suggestion: ${Suggestion.Content} `,
+				color: 'GREEN'
+			})
+		);
+	} else {
+		console.log('An error occured: could not find suggestion author in database');
```

Comment:
We don't need to log anything or need the else. The only reason we check is to make sure we don't get any issues if the user isn't cached.

Reply:
Removed :)